extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/VertxCoreRecorder.java

```diff
@@ -341,6 +345,16 @@ private static void setEventBusOptions(VertxConfiguration conf, VertxOptions opt
         options.setEventBusOptions(opts);
     }
 
+    private static void setAddressResolverOptions(VertxConfiguration conf, VertxOptions options) {
+        AddressResolverConfiguration ar = conf.resolver;
+        AddressResolverOptions opts = new AddressResolverOptions();
+        ar.cacheMaxTimeToLive.ifPresent(opts::setCacheMaxTimeToLive);
+        ar.cacheMinTimeToLive.ifPresent(opts::setCacheMinTimeToLive);
+        ar.cacheNegativeTimeToLive.ifPresent(opts::setCacheNegativeTimeToLive);
```

Comment:
We prefer to avoid lambdas in here for pure performance matters. And yes, I know it sucks :)

Reply:
Ok, I'll change it accordingly ...